### PR TITLE
Checked off my name and added username/emojis

### PR DESCRIPTION
**Pull request by dhruvkc**
Fulfilled assignment with username, checked box, and sweet emojis

Created a new pull request to the correct base repository (as per issue flagged by @AngusMaguir3 )

Please +1 if ready to commit:
@unakim 
@AngusMaguir3 
@lucassea
@stalerat

### CODE REVIEW

**Comment by unakim** [comment]
+1, Let's do this baby!

**Comment by AngusMaguir3** [comment]
+1, yea!

File: 2020_Fall/readme.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Make the Pull Request from your own GitHub account. Make sure you're changing th
 
 - [ ] Cerda, Daniel
 
-- [ ] Chandrasekhar, Dhruva
+- [X] Chandrasekhar, Dhruva (@dhruvakc) 🎠🎮🎃
```
Comment (joewadcan) [inline 2020_Fall/readme.md]: Hi @dhruvkc - looks like you've got a typo in your GitHub username. Please edit this existing Pull request to add another commit.

Comment (dhruvkc) [reply 2020_Fall/readme.md]: @joewadcan thank you for flagging, fixed the typo and added the commit!

**Comment by joewadcan** [comment]
Excellent, thanks!
